spec/system/subscription_spec.rb

```diff
@@ -0,0 +1,108 @@
+require 'rails_helper'
+
+describe 'Subscription plan' do
+  it 'view all subscriptions plans' do
+    Subscription.create!(name: 'Jornada Web com Rails',
+                         description: 'Esta assinatura irá englobar todos os cursos de Ruby e Rails',
+                         price: '50')
+    Subscription.create!(name: 'Jornada Mobile com Dart e Flutter',
+                         description: 'Esta assinatura irá englobar todos os cursos de Dart e o framework Flutter',
+                         price: '80')
+
+    visit subscriptions_path
+    # TODO: colocar um within aqui, ta dando bug por alguma razao
```

Comment:
deu certo essa aqui?

Reply:
Não funcionou, tava tentando fazer um 'div#id' mas não funcinou de jeito maneira. 

Sempre da esse erro:
 Failure/Error:
       within 'div#1' do
         expect(page).to have_text(subscription.name)
         expect(page).to have_text(subscription.description)
       end
     
     Selenium::WebDriver::Error::InvalidSelectorError:
       invalid selector: An invalid or illegal selector was specified
         (Session info: headless chrome=90.0.4430.72)

Reply:
Acho que é porque o id só é numérico coloca um teste antes que talvez dê certo.

Reply:
Não entendi, como assim?

Reply:
within 'div#1' você estaria chamando o ID e 1 é isso mesmo?

Reply:
Entendi! Só numerico não funciona, tive que colocar algum nome antes do numero